File: Desafios em C#/2164 - Fibonacci Rápido.cs
using System; 

//Developed by: @LucasMarcuzo

class URI {

    static void Main(string[] args) { 

        int n = int.Parse(Console.ReadLine());

        double result = ((Math.Pow(((1 + Math.Sqrt(5))/2), n) - Math.Pow(((1 - Math.Sqrt(5))/2), n)) / (Math.Sqrt(5)));

        Console.WriteLine("{0:F1}",result);

    }

}

File: Desafios em JavaScript/2164 - Fibonacci Rápido.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

var result = ((Math.pow(((1 + Math.sqrt(5))/2), n) - Math.pow(((1 - Math.sqrt(5))/2), n)) / (Math.sqrt(5)));

console.log(`${result.toFixed(1)}`);